package.json

```diff
@@ -30,29 +32,38 @@
     "@adobe/reactor-window": "*"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^17.0.0",
+    "@rollup/plugin-node-resolve": "^11.1.0",
     "camelize": "1.0.0",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
+    "husky": "^4.3.7",
     "inject-loader": "^4.0.1",
     "istanbul-instrumenter-loader": "^3.0.1",
     "jasmine-core": "^3.5.0",
-    "karma": "^5.0.3",
+    "karma": "^5.2.3",
     "karma-chrome-launcher": "^3.1.0",
     "karma-coverage": "^2.0.2",
-    "karma-firefox-launcher": "^1.3.0",
+    "karma-firefox-launcher": "^2.1.0",
     "karma-jasmine": "^3.1.1",
     "karma-jasmine-matchers": "^4.0.2",
     "karma-safari-launcher": "^1.0.0",
     "karma-sauce-launcher": "^4.1.4",
     "karma-webpack": "^4.0.2",
     "kebab-case": "1.0.0",
     "prettier": "^2.0.5",
-    "rollup": "^2.7.5",
-    "rollup-plugin-commonjs": "^10.1.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup": "^2.36.1",
     "uglify-js": "^3.9.4",
-    "webpack": "^4.43.0",
+    "webpack": "^4.46.0",
     "yargs": "^15.3.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm test"
```

Comment:
On a few other projects I did things a little different. I would run prettier and lint of pre-commit. And run the tests on pre-push: https://github.com/adobe/reactor-turbine-edge/blob/master/package.json#L23-L24.

Reply:
I saw the pattern that I used in other external projects, and I like it. Tests are usually slow to run. The prettier and eslint fix commands are run only on the staged files.